build.gradle

```diff
@@ -44,6 +44,9 @@ signing {
     sign configurations.archives
 }
 
+def sonatypeUsername = System.getenv('sonatypeUsername')
```

Comment:
Hey, sorry for the delay. I like the idea of checking in a gradle.properties with dummy values. Can you put a comment in the file saying from https://docs.gradle.org/current/userguide/userguide_single#properties

    # These values can be overridden via the .gradle/gradle.properties in the user's home directory